QUEST_LV_0200.tsv

```diff
@@ -2528,32 +2528,32 @@ QUEST_LV_0200_20150317_002527	Ris seems to have a problem. Find out what happene
-QUEST_LV_0200_20150317_002541	Ask Priest Raeli what to do with the branches.
-QUEST_LV_0200_20150317_002542	Defeat the demons and collect the ashes of the demons that were burnt
-QUEST_LV_0200_20150317_002543	Priest Raeli told you to use the branches to burn demons when their HP is below the half using the power of Goddess Austeja so you can get the ashes of the demons.
-QUEST_LV_0200_20150317_002544	You've collected the ashes of the demons. Talk to Priest Raeli.
-QUEST_LV_0200_20150317_002545	Priest Raeli made the marble using the ashes of the demons. Listen what to do with the marble.
-QUEST_LV_0200_20150317_002546	Recharge the marble of ashes with divine energies
-QUEST_LV_0200_20150317_002547	{memo X}The land where the Goddess Austeja used to control is full of divine magic energies. Go to Parama cliffs and recharge the marble of ashes with the energies.
-QUEST_LV_0200_20150317_002548	You've filled all the divine magic energies to the mable of ashes. Return to Priest Raeli.
-QUEST_LV_0200_20150317_002549	The divine magic energies gathered enough in the marble of ashes. Ask Priest Raeli what to do next with this.
-QUEST_LV_0200_20150317_002550	Restore the weak sealed tower
-QUEST_LV_0200_20150317_002551	Restore the weak sealed tower at Stula Highway. The demons may come to break the seal so you should hurry.
-QUEST_LV_0200_20150317_002552	Talk with Goddess Austeja
-QUEST_LV_0200_20150317_002553	As you restored the sealed tower which was changed by the demons, Goddess Austeja has appeared. Listen to the story of Goddess Austeja.
-QUEST_LV_0200_20150317_002554	Listen to the story of Goddess Austeja
-QUEST_LV_0200_20150317_002555	Goddess Austeja told you that the everything that is occurring at the apiary is the act of the demons that are trying to release the seal that is placed upon the Demon Lord.
-QUEST_LV_0200_20150317_002556	Priest Raeli is worrying about something. Ask him what to do.
+QUEST_LV_0200_20150317_002537	$It seems that Priest Raeli has something to say. Meet Priest Raeli.
+QUEST_LV_0200_20150317_002538	$Activate the Guardian Altar
+QUEST_LV_0200_20150317_002539	$Priest Raeli told you that more demons are incoming. Activate the guardian altar and stop the demons.
+QUEST_LV_0200_20150317_002540	$Stopped the monster attack. Return to Priest Raeli.
+QUEST_LV_0200_20150317_002541	$Ask Priest Raeli what to do with the branches.
+QUEST_LV_0200_20150317_002542	$Defeat the demons with the branches and collect the ashes of the demons
+QUEST_LV_0200_20150317_002543	$Priest Raeli told you to use the branches to burn demons when their HP is below half, using the power of Goddess Austeja, so you can get ashes from the demons.
+QUEST_LV_0200_20150317_002544	$You've collected the ashes of the demons. Talk to Priest Raeli.
+QUEST_LV_0200_20150317_002545	$Priest Raeli made the orb using the ashes of the demons. Listen what to do with the orb.
+QUEST_LV_0200_20150317_002546	$Recharge the orb made of ash with holy energy
```

Comment:
Unless the specific name of the item is orb made of ash, in which case I believe it should be changed like what I'm fixing to suggest, it should be "Ash Orb" or "Orb of Ash"